website/docs/docs/build/packages.md

```diff
@@ -143,6 +151,23 @@ To find the latest release for a package, navigate to the `Releases` tab in the
 
 As of v0.14.0, dbt will warn you if you install a package using the `git` syntax without specifying a version (see below).
 
+<VersionBlock firstVersion="1.4">
+
+### Internally hosted tarball URL
+
+Some organizations have security requirements to pull resources only from internal services. To address the need to install packages from hosted environments such as Artifactory or cloud storage buckets, internally hosted tarball URLs are supported for installing packages. 
```

Comment:
I'd like to clarify who is supporting what and make the sentence active so it will be easier to reduce errors when translated.
```suggestion
Some organizations have security requirements to pull resources only from internal services. To address the need to install packages from hosted environments such as Artifactory or cloud storage buckets, dbt Core enables you to install packages from internally-hosted tarball URLs. 
```